frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -694,7 +694,7 @@ def get_attachments(self):
 
 	def prepare_email_content(self):
 		email_account = self.get_outgoing_email_account()
-		if isinstance(self._bcc, list) and email_account.always_bcc:
+		if email_account.always_bcc:
 			self._bcc.append(email_account.always_bcc)
```

Comment:
We still have the problem that `self._bcc` could not be a list and then `.append` fails.

How about this:

```python
self._bcc = self.bcc + [email_account.always_bcc]
```

I think, this way we can preserve the current contents of `self.bcc` while also ensuring it's a list and our `always_bcc` is added always.